po/zh-CN.po

```diff
@@ -1889,13 +1889,13 @@ msgstr ""
 
 #: src/welcome-day-1.md:1
 msgid "# Welcome to Day 1"
-msgstr ""
+msgstr "# 欢迎来到第一天"
 
 #: src/welcome-day-1.md:3
 msgid ""
 "This is the first day of Comprehensive Rust. We will cover a lot of ground\n"
 "today:"
-msgstr ""
+msgstr "现在是学习Comprehensive Rust的第一天。今天我们会涉及很多内容："
```

Comment:
```suggestion
msgstr "现在是学习 Comprehensive Rust 的第一天。今天我们会涉及很多内容："
```

Reply:
I see spaces between Chinese and English in merged PRs. I guess this is the suggested style.